modules/apps/site-initializer/site-initializer-raylife/src/main/zippableResources/fragments/raylife/fragments/congrats-information/index.js

```diff
@@ -126,6 +167,11 @@ const main = async () => {
 			value: quoteComparison.moneyAndSecurities || false,
 		},
 	]);
+
+	if (!application.policySent) {
+		sendDigitalSignaturePolicy(application);
+		updateObjectPolicySent();
```

Comment:
It's look good!

@silasbsi in the future, would be better to use await for both, because it won't make sense to updateObjectPolicySent, if something goes wrong with sendDigitalSignaturePolicy.

It's ok for now, but we need to change this later

Reply:
Ok, Thanks. :smile: